### ISSUE TITLE
Custom dataset

### ISSUE THREAD

**Issue by tekrajchhetri** [ThiviyanThanapalasingam/relational-GraphConvolutionNetwork#10]
Could you provide some instructions on data preparation?

**Comment by zoubaihan** [comment]
> Could you provide some instructions on data preparation?

Hello，have you ever got the method how to build your own dataset? Could you please share me your method?

**Comment by thiviyanT** [comment]
I could add a paragraph to the README about using custom datasets for link prediction and node classification. Let me know if that would help.

**Comment by zoubaihan** [comment]
> I could add a paragraph to the README about using custom datasets for link prediction and node classification. Let me know if that would help.

Yeah, I think I really need it. Could you please add a paragraph to the README? I would be very appreciate it if you could add the paragraph about how to build our own dataset. Thanks!
